webui/react/src/components/kit/Select.tsx

```diff
@@ -110,13 +112,18 @@ const Select: React.FC<React.PropsWithChildren<SelectProps>> = forwardRef(functi
     return !!label && label.toLocaleLowerCase().includes(search.toLocaleLowerCase());
   }, []);
 
+  const getPopupContainer = (triggerNode: Element) => {
+    return (attachDropdownToContainer && triggerNode.parentElement) || document.body;
```

Comment:
nit: this reads oddly to me, i would prefer something like this: 
`return attachDropdownToContainer ? triggerNode.parentElement : document.body`